src/main/java/org/schemaspy/output/dot/schemaspy/DotOrphanProducer.java

```diff
@@ -0,0 +1,58 @@
+package org.schemaspy.output.dot.schemaspy;
+
+import org.schemaspy.model.Table;
+import org.schemaspy.output.dot.DotConfig;
+import org.schemaspy.output.dot.schemaspy.graph.Digraph;
+import org.schemaspy.output.dot.schemaspy.graph.Element;
+import org.schemaspy.output.dot.schemaspy.graph.EmptyGraph;
+import org.schemaspy.output.dot.schemaspy.graph.Graph;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DotOrphanProducer implements GraphProducer {
+
+    private final DotConfig dotConfig;
+    private final Collection<Table> tables;
+
+    public DotOrphanProducer(DotConfig dotConfig, Collection<Table> tables) {
+        this.dotConfig = dotConfig;
+        this.tables = tables;
+    }
+
+    public Graph graph() {
+        List<Table> orphans = orphans();
+        if (orphans.isEmpty()) {
+            return EmptyGraph.EMPTY;
+        }
+        return graph(orphans);
+    }
+    private List<Table> orphans() {
+        return tables
+                .stream()
+                .filter((table -> !table.isView()))
+                .filter(table -> table.isOrphan(false))
+                .collect(Collectors.toList());
+    }
+
+    private Graph graph(List<Table> orphans) {
+        return new Digraph(
+                () -> "orphans",
+                new DotConfigHeader(dotConfig, false),
+                orphans.stream()
+                        .sorted(Table::compareTo)
+                        .map(this::asDotNode)
+                        .toArray(Element[]::new)
+        );
+    }
+
+    private DotNode asDotNode(Table table) {
```

Comment:
Move to `Table`? I suspect we want to reuse this code.

Reply:
I don't think that table should know about DotNode.

But as so many times before, might be time for media/medium. Which probably would fit nice, since it could be used in the xml output.

But I would say new issue. This type of Table to DotNode exists in plenty of classes.

Reply:
Fair.